force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -81,7 +81,7 @@
 1. Open a third-party address verification account. (Some are free for nonprofits.)&lt;br/&gt;
 2. In your address verification service, create the required authentication ID and/or token.&lt;br/&gt;
 3. Return to this settings page, click the Edit button above, and paste the ID and/or token into the appropriate fields.&lt;br/&gt;
-4. Complete the remaining fields and click Save. The help text for each field provides more information. If you still need more help, see the &lt;a href=&quot;https://powerofus.force.com/NPSP_Address_Verification&quot;&gt;NPSP documentation&lt;/a&gt;.</value>
```

Comment:
The translation should also be updated as well. You can either remove the translated label or modify the translation if the change is straight forward.